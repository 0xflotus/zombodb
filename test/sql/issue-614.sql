select zdb.dump_query('idxevents', 'Field: 1');
select zdb.dump_query('idxevents', 'payload.Author: bob');