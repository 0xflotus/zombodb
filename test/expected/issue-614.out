select zdb.dump_query('idxevents', 'Field: 1');
     dump_query      
---------------------
 {                  +
   "match": {       +
     "Field": {     +
       "query": "1",+
       "boost": 1.0 +
     }              +
   }                +
 }
(1 row)

select zdb.dump_query('idxevents', 'payload.Author: bob');
          dump_query          
------------------------------
 {                           +
   "nested": {               +
     "path": "payload",      +
     "query": {              +
       "match": {            +
         "payload.Author": { +
           "query": "bob",   +
           "boost": 1.0      +
         }                   +
       }                     +
     },                      +
     "score_mode": "avg",    +
     "ignore_unmapped": false+
   }                         +
 }
(1 row)

